sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.JsonSchema/src/SchemaRegistryJsonEventSource.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.Tracing;
+using Azure.Core.Diagnostics;
+
+namespace Microsoft.Azure.Data.SchemaRegistry.JsonSchema
+{
+    /// <summary>
+    ///   Serves as an ETW event source for logging of information about
+    ///   Entity's client.
+    /// </summary>
+    ///
+    /// <remarks>
+    ///   When defining Start/Complete tasks, it is highly recommended that the CompleteEvent.Id must be exactly StartEvent.Id + 1.
```

Comment:
```suggestion
    ///   When defining Start/Complete tasks, it is highly recommended that the CompleteEvent.Id be exactly StartEvent.Id + 1.
```

If it's recommended, I don't think it's fair to call it a "must".